logging.properties

```diff
@@ -1,10 +1,16 @@
-handlers = java.util.logging.ConsoleHandler
 .level = INFO
-com.google.enterprise.adaptor.sharepoint.SharePointAdaptor.level = FINER
-com.google.enterprise.adaptor.sharepoint.SharePointUserProfileAdaptor.level = FINER
-com.google.enterprise.adaptor.level = FINER
+com.google.enterprise.adaptor.sharepoint.level = ALL
+#com.google.enterprise.adaptor.sharepoint.SharePointUserProfileAdaptor.level = FINER
```

Comment:
I don't see this making sense, certainly not by itself. I think the package level alone is fine.